demos/common/transport/transport_tls_socket_using_mbedtls.c

```diff
@@ -840,8 +840,8 @@ int32_t TLS_Socket_Recv( NetworkContext_t * pxNetworkContext,
     }
     else if( lTLSStatus < 0 )
     {
-        LogError( ( "Failed to read data: mbedTLSError= %s : %s.",
-                    mbedtlsHighLevelCodeOrDefault( lTLSStatus ),
+        LogError( ( "Failed to read data: mbedTLSError[%d]= %s : %s.",
+                    lTLSStatus, mbedtlsHighLevelCodeOrDefault( lTLSStatus ),
```

Comment:
Nit - why is this function this called `lTLSStatus` but other places its `lMbedtlsError`?  It seems it should be consistent??

Reply:
Ack